controller/tenant_test.go

```diff
@@ -194,11 +194,11 @@ func (s *TenantControllerTestSuite) TestSetupTenantOKWhenNoTenantExistsInParalle
 				run.Wait()
 
 				// when
-				err = ctrl.Setup(setupCtx)
+				ctrl.Setup(setupCtx)
```

Comment:
LHS is missing and sub-sequent checks will be wrong.

Reply:
good catch, thanks. fixed 8d16c1c